package.json

```diff
@@ -16,6 +16,9 @@
 		"test:js": "wp-scripts test-unit-js"
 	},
 	"devDependencies": {
+		"@testing-library/jest-dom": "^5.16.4",
+		"@testing-library/react": "^13.0.1",
+		"@testing-library/user-event": "^14.1.0",
```

Comment:
It looks like these are already referenced in `package-lock.json`, so there's no `package-lock.json` diff to commit.